pkg/cli/admin/release/info.go

```diff
@@ -497,6 +500,7 @@ func (o *InfoOptions) Run() error {
 			fmt.Fprintf(o.ErrOut, "error: %v\n", err)
 			continue
 		}
+		o.Architecture[image] = release.Config.Architecture
```

Comment:
What is the benefit to storing this here, after we'e already successfully called `o.describeImage(release)`?

Reply:
Ah, for the `infoOpts.Run` call in `mirror.go`, when we are loading from a pullspec.  I'm... not thrilled about that, but it's an existing approach, so we should not block this PR on cleaning up that API.

Reply:
[This][1] might be one way to avoid it.

[1]: https://github.com/openshift/oc/pull/646#discussion_r527400619